MekHQ/src/mekhq/campaign/CampaignOptions.java

```diff
@@ -2205,6 +2243,249 @@ public void setAgeRangeRandomDeathFemaleValues(final Map<TenYearAgeRange, Double
         this.ageRangeRandomDeathFemaleValues = ageRangeRandomDeathFemaleValues;
     }
     //endregion Death
+
+    //region Awards
+    /**
+     * @return whether to issue awards to dead personnel
+     */
+    public boolean isIssuePosthumousAwards() {
```

Comment:
I believe the code style recommends using 'getX' and 'setX' for getters and setters of private members; even if not, 'getX'/'setX' is the standard nomenclature.

Reply:
The style guide refers to "getters and setters", so I would go with 'getX' / 'setX', although apparently a '90s-era JavaBeans style guide also allows isX for getting `boolean` values.

I guess if we have a mix of getX / isX in other parts of the code, there's no harm having it here.